### DISCUSSION TITLE
CNAME file deleted by GitHub Actions publishing workflow

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [squidfunk/mkdocs-material#4290] [Q&A] [answered]
I've just added a Custom Domain to my GitHub Pages site. This adds a CNAME file to the repository. When the [Insiders GitHub Pages Action Workflow](https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions) runs, it deletes the CNAME file and I have to manually re-add it. 

Is there a command that can be added to the workflow that will preserve the CNAME file? If so, this may be a useful addition to the documentation.

**Comment by squidfunk** [comment, marked as answer]
Good question. I've never used CNAMEs. This might be related: https://github.com/mkdocs/mkdocs/issues/2237

**Reply by Stephen-Gates** [reply]
Thank you. https://github.com/mkdocs/mkdocs/issues/2237 is the answer. Simply add your CNAME file in /docs.  🙏
